extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -51,12 +51,13 @@ export class LocalSubscriptionPlatform {
    * @return {!Promise<!Entitlements>}
    */
   getEntitlements() {
-    user().assert(this.serviceConfig_['paywallUrl'],
-        'Service config does not have paywall Url');
+    const authUrl = user().assert(this.serviceConfig_['authorizationUrl'],
```

Comment:
It's best to initialize all `authorizationUrl` in the constructor and also call `assertHttpsUrl` on it.

Reply:
Done